src/main/java/discord4j/discordjson/json/ApplicationCommandOptionData.java

```diff
@@ -0,0 +1,53 @@
+package discord4j.discordjson.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import discord4j.discordjson.possible.Possible;
+import org.immutables.value.Value;
+
+import java.util.List;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableApplicationCommandOptionData.class)
+@JsonDeserialize(as = ImmutableApplicationCommandOptionData.class)
+public interface ApplicationCommandOptionData {
+
+    static ImmutableApplicationCommandOptionData.Builder builder() {
+        return ImmutableApplicationCommandOptionData.builder();
+    }
+
+    /**
+     * value of ApplicationCommandOptionType
+     */
+    int type();
+
+    /**
+     * 1-32 character name
+     */
+    String name();
+
+    /**
+     * 1-100 character description
+     */
+    String description();
+
+    /**
+     * the first required option for the user to complete -- only one option can be default
+     */
+    @JsonProperty("default")
+    Possible<Boolean> isDefault();
```

Comment:
It should be `default()` to be consistent with the rest of the API. For example, just below, `required()`

Reply:
Oh, nevermind, I forgot default is a reserved name